bot/tests/test_docupload_task.py

```diff
@@ -34,7 +34,7 @@
 
 - file [docs/folderB/folderC/index.rst](http://firefox-test-docs.mozilla.org/section2/index.html)
 
-- file [docs/folderB/folderC/subfolder/index.rst](http://firefox-test-docs.mozilla.org/section2/subfolder/index.html)
+- file [docs/folderB/folderC/subfolder/index.md](http://firefox-test-docs.mozilla.org/section2/subfolder/index.html)
```

Comment:
Nit: why changing from .rst to .md?

Reply:
It's just to assert that we properly handle `.md` files too, there wasn't one in the test before.